mobile/src/bundles/common/components/slider/styles.ts

```diff
@@ -3,6 +3,7 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
     slider: {
         borderRadius: 6,
+        marginHorizontal: -10,
```

Comment:
again why -? And can we pass it via containerStyle?

Reply:
Let's keep everything without margin: -value for now, we'll fix it later if needed

Reply:
done

Reply:
<img width="392" alt="Screenshot 2023-08-29 at 12 53 18" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/70234990/0262aded-93d2-4343-87da-ef5bd04bfcc3">